src/pages/Menu.jsx

```diff
@@ -62,21 +61,40 @@ const Menu = () => {
         ))}
       </Tabs>
 
-      <Grid container spacing={4.5}>
-        {categorizedDishes?.map((d) => (
-          <Grid item key={d.id} xs={6} sm={4} md={3}>
```

Comment:
The problem with this is that `categorizedDishes` is initialized with data fetched by RTK query, but there will be a loading phase before the initialization. Since there is no UI handling the loading phase, `categorizedDishes` will be set to [] initially. When user clicks on Menu in the navbar for the first time, there wont be any data rendered.